### ISSUE TITLE
Cannot use env APP_LOCALE as locale

### ISSUE THREAD

**Issue by rzeczkowskip** [kucharovic/money-bundle#15]
When using env var APP_LOCALE as locale in app, the bundle's extension throws an exception:

`Locale '%env(APP_LOCALE)%' is not valid.`

```
at Configuration->__construct('env_APP_LOCALE_2bdb911ae78a24b20cbc0b3a32d66a9e')
in JKMoneyExtension.php (line 24)
```

Easiest fix would be hard-coding some default currency instead of trying to get code from NumberFormatter.

I can submit a PR if required

**Comment by enumag** [comment]
Yeah. Using DIC parameters in `Extension::load()` is too early since they are not fully resolved yet.
